packages/integration/api-report.md

```diff
@@ -187,6 +191,7 @@ export type BitbucketCloudIntegrationConfig = {
   username?: string;
   appPassword?: string;
   token?: string;
+  throttling?: ThrottlingConfig;
```

Comment:
I really like this approach, but in line with @benjdlambert suggestion of making this more consistent across services, I am curious if perhaps we put the `throttling` key underneath of a `fetch` key, like so:

```
fetch:
  throttling:
    count: 1
    interval: ...
```

My reasoning is that we may want to add more properties in the future to configure the FetchService and having a consistent parent key across all plugins would be nice as an indicator to developers that a section of the config contains certain properties and uses the FetchService.

Reply:
added